doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+    * :ref:`gps_with_supl_support_sample` - added
+    * :ref:`https_client` - added
+    * :ref:`serial_lte_modem` - added
+    * :ref:`nrf_coap_client_sample` - updated
+
+  * Bluetooth Low Energy:
+
+    * :ref:`ble_llpm` - added ??? not mentioned in release notes
+    * :ref:`bluetooth_mesh_light` - added ??? not mentioned in release notes
+    * :ref:`bluetooth_mesh_light_switch` - added ??? not mentioned in release notes
+    * :ref:`peripheral_gatt_dm` - added ??? not mentioned in release notes
+    * :ref:`ble_throughput` - updated
+
+  * NFC:
+
+    * :ref:`nrf-nfc-system-off-sample` - added ??? not mentioned in release notes
+    * :ref:`nfc_tnep_poller` - added ??? not mentioned in release notes
+    * :ref:`nfc_tnep_tag` - added ??? not mentioned in release notes
+
+  * Other:
+
+    * :ref:`radio_test` - added ??? not mentioned in release notes
+    * :ref:`timeslot_sample` - added ??? not mentioned in release notes
+    * :ref:`bootloader` - updated
+    * :ref:`nrf_desktop` - updated
+
+
+* Added or updated documentation for the following libraries:
+
+  * nRF9160:
+
+    * :ref:`lib_aws_iot` - added
+    * :ref:`cloud_api_readme` - added
+    * :ref:`modem_key_mgmt` - added
+    * :ref:`sms_readme` - added
+    * :ref:`supl_client` - added
+    * :ref:`at_cmd_readme` - updated
+    * :ref:`at_cmd_parser_readme` - updated
+    * :ref:`lib_download_client` - updated
+
+  * Bluetooth Low Energy:
+
+    * :ref:`bt_mesh_dk_prov` - added
+    * :ref:`latency_c_readme` - added
+    * :ref:`latency_readme` - added
+    * :ref:`shell_bt_nus_readme` - updated
+
+  * NFC:
+
+    * :ref:`nfc_t4t_cc_file_readme` - added
+    * :ref:`st25r3911b_nfc_readme` - added
+    * :ref:`nfc_t4t_hl_procedure_readme` - added
+    * :ref:`tnep_poller_readme` - added
+    * :ref:`tnep_tag_readme` - added
+
+  * Other:
+
+    * :ref:`doc_bl_crypto` - added
+    * :ref:`doc_bl_validation` - added
+    * :ref:`fprotect_readme` - added
+    * :ref:`lib_dfu_target` - updated
+    * :ref:`doc_fw_info` - updated
+    * :ref:`partition_manager` - updated
+
+
+* Added or updated the following documentation:
+
+  * Getting started:
+
+    * :ref:`gs_installing` - restructured parts of the content
+    * :ref:`gs_programming` - restructured the content and added information about building on the command line
+    * :ref:`gs_modifying` - updated the content and added information about configuring an application
+
+  * User guides:
+
+    * :ref:`ug_nrf5340` - added
+    * :ref:`ug_thingy91` - added
+    * :ref:`ug_ble_controller` - added
+    * :ref:`ug_multi_image` - updated with content that was removed from the Zephyr fork
+
+  * nrfxlib:
+
+    * :ref:`nrfxlib:ble_controller_readme` - updated to match current version of the nRF Bluetooth LE Controller
+    * :ref:`nrfxlib:bsdlib` - extended and restructured the content
+    * :ref:`nrfxlib:mpsl` - added
+
+
+
+Known issues
+************
+
+??? Must be checked and updated
+
+nRF9160
+=======
+
```

Comment:
* In the time window between a PSM request and a successful response, sending LTE packets can cause the send() function in bsdlib to hang for long periods. (from seconds to tens of minutes) Behavior depends on cellular network connection.  

@jtguggedal  or @evenl please update if you have a better suggestion to text.

Reply:
send() on a socket.

PSM = Power saving mode

Reply:
The originial description was not correct. Here is a new proposal.  "nrf_send() can be blocking for several minutes even when socket is configured for non blocking operation" @jtguggedal @evenl please comment.

Reply:
I think it's correct to say that a `send()` call can block even though it's configured to be non-blocking.

Reply:
Agree.

Reply:
But I don't have detailed knowledge of bsdlib's inner workings, so can't really say what is the "most correct" phrasing here

Reply:
Updated to:
* The :cpp:func:`nrf_send` function in the :ref:`nrfxlib:bsdlib` might be blocking for several minutes, even when the socket is configured for non-blocking operation.
  The behavior depends on the cellular network connection.